docs/DeveloperGuide.md

```diff
@@ -260,6 +261,46 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Set/Unset payment made feature
+
+#### Implementation
+
+The purpose of the set/unset payment made feature is for tutors to modify a student's payment status.
+
+This feature implements the following operations:
+
+* `PaidCommand#execute()` - Updates the payment status of the student to paid.
+* `UnpaidCommand#execute()` - Updates the payment status of the student to unpaid.
+
+This feature is facilitated by the following operations:
+
+* `TutorAidParser#parseCommand()` - Calls `PaidCommandParser#parse()` or `UnpaidCommandParser#parse()` with the specified student index, depending on the command word in the user input.
+* `PaidCommandParser#parse()` - Returns an instance of `PaidCommand`.
+* `UnpaidCommandParser#parse()` - Returns an instance of `UnpaidCommand`.
+
+To represent a student's payment status, a `PaymentStatus` class is created. It stores an immutable instance variable `hasPaid`, of boolean type. We then work with the `Student` model, and implement it as a field in Student.
```

Comment:
Can consider rephrasing this, as the 'it' reads as if the field is for the Model.